config/prow/config.yaml

```diff
@@ -143,6 +143,8 @@ deck:
         name: podinfo
       required_files:
         - ^podinfo\.json$
+      optional_files:
+        - ^prowjob\.json&
```

Comment:
```suggestion
        - ^prowjob\.json$
```

Reply:
We should update the config in a follow up PR after the source is deployed. This is a breaking change to old versions of the lens because they assume a single artifact file and that the file is a podinfo.json.

Reply:
good point, deleted